rust/tools/authorization-logic/src/test/test_claim_importing.rs

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 The Raksha Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#[cfg(test)]
+mod test {
```

Comment:
These tests should probably all be in a tests dir, alongside src: https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-directory

Reply:
That makes sense. Getting this to build again after doing that involves doing stuff, and so there is an issue to track this [here](https://github.com/google-research/raksha/issues/17)